gr-uhd/python/uhd/bindings/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ list(APPEND uhd_python_files
     usrp_block_python.cc
     usrp_sink_python.cc
     usrp_source_python.cc
+    rfnoc_graph_python.cc
```

Comment:
does rfnoc_rx,tx_streamer need to go here as well?

Reply:
yes!